python/lib/dependabot/python/version.rb

```diff
@@ -66,6 +66,10 @@ def <=>(other)
         local_version_comparison(other)
       end
 
+      def truncate_to_major_minor
```

Comment:
This could be moved all the way to `Dependabot::Version` as we could probably use it elsehwere, but I thought that might be a little heavy-handed given how much version-like strings vary across ecosystems... So for now sticking with Python.